index.md

```diff
@@ -52,3 +203,61 @@ which most in the community are members of.
 If you have technical questions around DNS flag day you can join
 [the DNS-operations mailing list](https://lists.dns-oarc.net/mailman/listinfo/dns-operations)
 and ask them there.
+
+Get in touch
+============
+
+For press & media inquiries please use media (at) dns-oarc.net and please put
+“DNS Flag Day" in the email subject line.
+
+- Web https://dnsflagday.net/
+- Twitter https://twitter.com/dnsflagday
+- Announcements: https://lists.dns-oarc.net/mailman/listinfo/dns-announce
+- Discussion: https://lists.dns-oarc.net/mailman/listinfo/dns-operations
+
+FAQ
+===
+
+- Q: Is DNS over UDP dead?
+
+  A: No, UDP will still be the main means of transportation.
+
+- Q: TL;DR [RFC 7766](https://tools.ietf.org/html/rfc7766)
+
+  A: DNS **MUST** work over TCP!
+
+- Q: Will everything break on date-to-be-decided 2020?
+
+  A: Certainly not everything! Only a small percentage of sites is affected, and this number is shrinking as operators
+     work on fixing their systems. On the date that will be announced major DNS resolver operators will stop
+     tolerating misbehavior which breaks published standards, so this change will not affect sites which follows
+     published standards. Also, on the announced date software vendors will change behavior
+     _**in new software releases**_, so this change will also slowly affect others who operate their own
+     DNS resolvers.
+
+- Q: Why is TCP support so important?
+
+  A: DNS software should implement TCP support from the beginning, although some
```

Comment:
A: Blocking TCP, or failure to support TCP, may result in resolution failure and application-level timeouts.  

In addition, TCP can be operated in a mode that does not require IP fragmentation, and makes it harder to spoof DNS responses.

It was expected, but not explicitly required from the early standards specification that DNS should work over TCP. Hence ten years ago [RFC 5966](https://tools.ietf.org/html/rfc5966) made it clear that TCP support mandatory for compliance with the Internet standards for DNS.

Reply:
@vrisk are you suggesting to review the whole answer or parts of it?

@pspacek @matje Can't remember who of you wrote the answer but are you OK with the change if it changes the whole answer?

Reply:
@jelu I took the first stab writing an answer. I am okay if the text that @vrisk suggests changes my whole answer: It covers everything I said before.

Some minor suggestions: "It was expected" sounds a bit weak, and I don't see the need for "IS" being in all capitals. I propose to change the last paragraph that Vicky suggested to:
```
It was recommended from the early standards specification that DNS should work over TCP.  Some implementers may have taken that to mean TCP was optional, and so ten years ago [RFC 5966](https://tools.ietf.org/html/rfc5966) made it clear that TCP support is absolutely required for compliance with the Internet standards for DNS.
```

Reply:
yeah, I was trying to replace the answer. but actually rereading it, I don't think my answer was so great either.  "TCP can be operated in a mode" - what the heck does that even mean?

Reply:
"TCP can be operated in a mode" can be more clear (but should it for a FAQ?). The specifics:  they normally implement Path MTU Discovery and can avoid IP fragmentation of TCP segments.

I don't think we should make the perfect text, but focus on what should be in the answer. I do appreciate the review to improve the answer. I think the order of arguments that @vrisk proposes also makes more sense. How about:

```suggestion
  A: Blocking TCP, or failure to support TCP, may result in resolution failure and application-level timeouts. 
  
  Furthermore, TCP normally implements Path MTU Discovery and can avoid IP fragmentation of TCP segments. It also makes it harder to spoof DNS responses.
  
  Finally, TCP support was recommended from the early standards specification, but some implementers may have taken that to mean TCP was optional, and so ten years ago RFC 5966 made it clear that TCP support is absolutely required for compliance with the Internet standards for DNS.
```

Reply:
```suggestion
  A: Blocking TCP, or failure to support TCP, may result in resolution failure and
     application-level timeouts.

     Furthermore, TCP normally implements Path MTU Discovery and can avoid
     IP fragmentation of TCP segments. It also makes it harder to spoof DNS responses.

     Finally, TCP support was recommended from the early standards specification, but
     some implementers may have taken that to mean TCP was optional, and so
     ten years ago (August 2010) [RFC 5966](https://tools.ietf.org/html/rfc5966)
     made it clear that TCP support is absolutely required for compliance with the
     Internet standards for DNS.
```